tests/integration/cli/projects/script/scripts/deploy.py

```diff
@@ -1,6 +1,11 @@
 from ape import networks
 
+# Make sure the re-import bug goes away
+from ._utils import deploy  # noqa: F401
+
 
 def main():
-    assert networks.active_provider.name == "test"
-    print("Script ran!")  # noqa: T001
+    with networks.parse_network_choice("::test"):
```

Comment:
I don't think this part is needed anymore if `--network` is automatically passed in?
I am not sure how that would affect the test though